packages/terra-date-picker/src/DateRange.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { PropTypes } from 'react';
+import DatePicker from '../lib/DatePicker';
+
+const propTypes = {
+  endDate: PropTypes.oneOfType([PropTypes.object]),
+  startDate: PropTypes.oneOfType([PropTypes.object]),
+  onChange: PropTypes.func,
+};
+
+class DateRange extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      startDate: props.startDate,
+      endDate: props.endDate,
+    };
+    this.handleChangeStart = this.handleChangeStart.bind(this);
+    this.handleChangeEnd = this.handleChangeEnd.bind(this);
+  }
+
+  handleChange({ startDate, endDate }) {
```

Comment:
If you feel like it, this function can use default props: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters